bin/retrieve_eload_from_lts.py

```diff
@@ -0,0 +1,149 @@
+        os.makedirs(eloads_output_path)
+
+    logging.info(f"Retrieving Eloads")
+    for eload_archive_info in eloads_archive_info['eload_archives']:
+        ## Retrieve eload
+        eload_name = eload_archive_info['name']
+        eload_archive_path = os.path.join(eloads_archive_dir, eload_name)
+        files_dirs_to_retrieve = get_files_or_dirs_to_retrieve_from_archive(eload_archive_info)
+        retrieve_archive(eload_archive_path, eloads_output_path, files_dirs_to_retrieve)
+
+        # Uncompress files
+        files_to_uncompress = get_compressed_files_in_dirs(
+            os.path.join(eloads_output_path, eload_name.rstrip('.tar')))
+        uncompress_files(files_to_uncompress)
+
+        # Retrieve associated project if specified
+        if eload_archive_info['retrieve_associated_project']:
+            project_acc = get_project_from_eload_config(eloads_output_path, eload_name)
+            if project_acc:
+                retrieve_archive(os.path.join(projects_archive_dir, f'{project_acc}.tar'), projects_output_path)
+                project_files_to_uncompress = get_compressed_files_in_dirs(os.path.join(projects_output_path, project_acc))
+                uncompress_files(project_files_to_uncompress)
+
+        # Update noah paths to codon in eload config
+        if eload_archive_info['update_noah_paths']:
+            update_path_in_eload_config(eloads_output_path, eload_name)
+
+
+def retrieve_projects(project_archive_info, projects_archive_dir):
+    projects_output_path = project_archive_info['projects_output_path']
+
+    if not os.path.exists(projects_output_path):
+        os.makedirs(projects_output_path)
+
+    logging.info(f"Retrieving Projects")
+    for project_archive_info in project_archive_info['project_archives']:
+        # Retrieve Project
+        project_name = project_archive_info['name']
+        project_archive_path = os.path.join(projects_archive_dir, project_name)
+        files_dirs_to_retrieve = get_files_or_dirs_to_retrieve_from_archive(project_archive_path)
+        retrieve_archive(project_archive_path, projects_output_path, files_dirs_to_retrieve)
+
+        # Uncompress files
+        files_to_uncompress = get_compressed_files_in_dirs(
+            os.path.join(projects_output_path, project_name.rstrip('.tar')))
+        uncompress_files(files_to_uncompress)
+
+
+def retrieve_eloads_and_projects(config_file):
+    if os.path.isfile(config_file):
+        with open(config_file, 'r') as f:
+            config = yaml.safe_load(f)
+
+            for archive_type in config['retrieve']:
+                if archive_type == 'eloads':
+                    retrieve_eloads(config['retrieve']['eloads'], config['retrieve']['eloads_archive_dir'],
+                                    config['retrieve']['projects_archive_dir'])
+                elif archive_type == 'projects':
+                    retrieve_projects(config['retrieve']['projects'], config['retrieve']['projects_archive_dir'])
```

Comment:
I'm not sure about the `retrieve` keyword in the config file. I think we might want to used the same paths for archiving and it might make things complicated.
Maybe this is something we should have a quick discussion about.